UsingSimpleSqlProvider/app/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion compileSdkVer
+    buildToolsVersion buildToolsVer
+    defaultConfig {
+        applicationId "me.gurpreetsk.usingsimplesqlprovider"
+        minSdkVersion 15
```

Comment:
reference these values from dependencies.gradle file. See other examples for references